index.html

```diff
@@ -81,250 +83,65 @@
-              </button>
-            </div>
-          </li>
-
-          <li>
-            <div class="card card-vertical width-25">
-              <div class="card-vertical-header">
-                <span class="status-badge text-center width-6p5"
-                  >4+ <i class="fas fa-star"></i
-                ></span>
-
-                <button class="bg-red-50 card-icon-btn margin-l-auto">
-                  <i class="fas fa-heart margin-auto"></i>
-                </button>
-              </div>
-
-              <img
-                class="card-vertical-img"
-                src="assets/jpegs/marvel-tshirt.jpg"
-                alt="marvel t-shirt for man jpg"
-              />
-
-              <h1 class="card-head height-3p5">
-                Marvel Men's Regular Fit T-Shirt
-              </h1>
-              <p class="card-text height-7"></p>
+      <section class="d-flex padding-3 section-container">
+        <div class="card flex-direction-col margin-auto width-33">
+          <img
+            class="card-vertical-img height-12"
+            src="assets/jpegs/dr-strange.jpg"
+            alt="unsplash dr. strange jpg"
+          />
 
-              <div class="card-vertical-footer margin-t-1">
-                <span class="current-price">
-                  <strong>₹379</strong>
-                </span>
-                <span class="earlier-price">₹899</span>
-                <span class="pct-off"><strong>58% off</strong></span>
-              </div>
+          <form class="border-r-unset form padding-1-2-3 max-width-100pct">
+            <h1 class="margin-b-2 text-slate-600">Log In</h1>
+
+            <label class="font-weight-bold" for="email-id">Email Address</label>
+            <input
+              class="bg-unset border-r-unset input margin-lr-0 width-100pct"
+              type="email"
+              name="email-id"
+              id="email-id"
+              placeholder="inclusive.mart@example.com"
+            />
+            <label class="font-weight-bold" for="password">Password</label>
+            <input
+              class="bg-unset border-r-unset input margin-lr-0 width-100pct"
+              type="password"
+              name="password"
+              id="password"
+              placeholder="********"
+            />
+
+            <div class="align-i-ctr d-flex justify-c-sb margin-b-1 margin-t-2">
+              <label class="cursor-ptr font-size-1p5" for="remember-me">
+                <input
+                  class="cursor-ptr"
+                  type="checkbox"
+                  name="remember-me"
+                  id="remember-me"
+                />
+                Remember me
+              </label>
 
-              <button
-                class="bg-add-cart bg-add-cart-hover btn font-weight-bold text-slate-800"
+              <a
+                class="font-size-1p5 font-weight-bold text-slate-600 text-slate-800-hover"
+                href="#"
+                >Forgt Password?</a
```

Comment:
Type error. It should be forgot

Reply:
Thanks, Sainath. I was changing it from `Forget` to `Forgot`, and then this happened 😅